### ISSUE TITLE
support styling top-level element with :host 

### ISSUE THREAD

**Issue by ahdinosaur** [stackcss/postcss-prefix#2]
from https://github.com/sheetify/sheetify/issues/22#issuecomment-160019440,

``` css
/* This: */
:host { color: green; }
:host(:hover) { color: blue; }
/* Becomes: */
.prefix { color: green; }
.prefix:hover { color: blue; }
```

**Comment by yoshuawuyts** [comment]
This should be fixed by #3. Released as `v1.0.2`

**Comment by yoshuawuyts** [comment]
I just realized how good this is - thanks @ahdinosaur 👌
